composer/src/composer.rs

```diff
@@ -549,6 +557,10 @@ pub struct Builder {
     shutdown_order: Vec<ContainerName>,
     /// the network used by this experiment
     network: Ipv4Network,
+    /// network mode for the test.
```

Comment:
Ah we had network here already, we couldn't make use of that as host?

Reply:
I noticed it, but I don't think it's of any use for host network mode. To use it we have to set ip range in it and then that's used to configure IPAM, we don't need all those.